.github/infrastructure/conformance/azure/conf-test-azure-eventhubs.bicep

```diff
@@ -61,6 +85,39 @@ resource eventHubsNamespace 'Microsoft.EventHub/namespaces@2017-04-01' = {
       name: eventHubPubsubConsumerGroupName
     }
   }
+  resource certificationEventHubPubsub1 'eventhubs' = {
+    name: certificationEventHubPubsub1Name
+    properties: {
+      messageRetentionInDays: 1
+    }
+    resource certificationEventHubPubsub1Policy 'authorizationRules' = {
+      name: certificationEventHubPubsub1PolicyName
+      properties: {
+        rights: [
+          'Send'
+          'Listen'
+        ]
+      }
+    }
+    resource eventHubPubsubConsumerGroup1 'consumergroups' = {
+      name: certificationConsumerGroupName1
+    }
+    resource eventHubPubsubConsumerGroup2 'consumergroups' = {
+      name: certificationConsumerGroupName2
+    }
+  }
+  resource certificationEventHubPubsub2 'eventhubs' = {
```

Comment:
You never seem to use this. Given that, can you please remove this and rename `Pubsub1` to something more meaningful?

Especially where the variable is exported - can you name it `AzureEventHubsPubsubCertificationTopicConnectionString` or something along these lines? We already have enough topic used for conformance so it's confusing to have a random Pubsub1.. but there is no Pubsub2 being used.

Reply:
`certificationEventHubPubsub2` is being leveraged as part of following tests

- Publishing to multiple topics from the component with namespace level access.
- Publishing to a topic which is not being subscribed by any consumer.

Good point on naming convention, let me give it a shot.